test/test_pebble.py

```diff
@@ -610,6 +639,110 @@ def test_service_equality(self):
         }
         self.assertEqual(plan.services, services_as_dict)
 
+    def test_plan_equality(self):
+        plan1 = pebble.Plan('''
+services:
+  foo:
+    override: replace
+    command: echo foo
+''')
+        self.assertFalse(plan1 == "foo")
+        plan2 = pebble.Plan('''
+services:
+  foo:
+    command: echo foo
+    override: replace
+''')
+        self.assertTrue(plan1 == plan2)
+        plan1_as_dict = pebble.PlanDict({
+            "services": {
+                "foo": pebble.ServiceDict({
+                    "command": "echo foo",
+                    "override": "replace",
+                })
+            },
+        })
+        self.assertTrue(plan1, plan1_as_dict)
+        plan3 = pebble.Plan('''
+services:
+  foo:
+    override: replace
+    command: echo bar
+''')
+        self.assertFalse(plan1 == plan3)
+        plan4 = pebble.Plan('''
+services:
+ foo:
+  override: replace
+  command: echo foo
+
+checks:
+ bar:
+  http:
+   https://example.com/
+
+log-targets:
+ baz:
+  override: replace
+  type: loki
+  location: https://example.com:3100/loki/api/v1/push
+''')
+        plan5 = pebble.Plan('''
+services:
+ foo:
+  override: replace
+  command: echo foo
+
+checks:
+ bar:
+  http:
+   https://example.org/
+
+log-targets:
+ baz:
+  override: replace
+  type: loki
+  location: https://example.com:3100/loki/api/v1/push
+''')
+        self.assertFalse(plan4 == plan5)
```

Comment:
It wasn't at all obvious to me why this was false (finally found the .com vs .org). Can we comment here or above?